src/server/api/controllers/channels-message-controller.js

```diff
@@ -15,27 +15,29 @@ const getChannelsId = async (userId) => {
       channels.map(async (element) => {
         const message = await knex('channel_messages')
           .select(
-            'message',
+            'channel_messages.message',
             'channel_messages.updated_at as updatedAt',
-            'channel_messages.fk_channel_id',
-            'title',
+            'channels.id as channelId',
+            'channels.title',
+            'channel_members.fk_user_id as userId',
           )
-          .innerJoin(
-            'channels',
-            'channels.id',
-            '=',
-            'channel_messages.fk_channel_id',
-          )
-          .where('fk_channel_id', '=', element.fk_channel_id)
-          .orderBy('channel_messages.updated_at', 'asc')
+          .innerJoin('channels', {
+            'channel_messages.fk_channel_id': element.channelId,
+          })
+          .where('channels.id', element.channelId)
+          .innerJoin('channel_members', {
+            'channel_messages.fk_channel_id': element.channelId,
+          })
+          .where('channel_members.fk_channel_id', element.channelId)
```

Comment:
This `where` is not necessary since `innerJoin` will already do that for you.

Reply:
it is necessary because if I don't have WHERE I receive channel.title for channel with id 1.